framework/src/components/logger/logger.js

```diff
@@ -58,7 +65,7 @@ const createLogger = ({ fileLogLevel, consoleLogLevel, logFileName }) => {
 			: [];
 	const streams = [...consoleStream, ...fileStream];
 	return bunyan.createLogger({
-		name: 'lisk-framework',
+		name,
```

Comment:
This seems to follow right pattern as of Bunyan documentation. But I suggest another path. I suggest that logger name should be fixed `Lisk-framework` for all logger instances. The extra attribute we are trying to set should be set as `module: 'lisk-chain'` or  `module: 'lisk-network'` etc. Which is called bounded fields in terms of Bunyan. 


Having fixed static name for all log entries is very essential. Think of a fact that one have a log analysis service e.g. Logstash where he redirects all logs from many other products. And now if he want to find all log entries from Lisk, he can find it only by having static name `lisk-framework` in all entries.